q2_vsearch/plugin_setup.py

```diff
@@ -283,11 +283,11 @@
             0, 8, inclusive_start=True, inclusive_end=True)
     },
     outputs=[
-        ('joined_sequences', SampleData[JoinedSequencesWithQuality])
+        ('merged_sequences', SampleData[JoinedSequencesWithQuality])
```

Comment:
See discussion of the name of this semantic type.
https://github.com/qiime2/q2-vsearch/issues/84#issuecomment-1249606494